general/administration/hardware-acceleration.md

```diff
@@ -47,17 +46,37 @@ or using `lshw`:
 lshw -C display
 ```
 
+## Tips on H.264 / AVC 10-bit videos
+
+The hardware decoding of H.264 10-bit aka High10 profile video is not supported by any Intel, AMD or NVIDIA GPU.
+
+Please consider upgrading these videos to HEVC 10-bit aka Main10 profile if you want to offload your CPU usage during transcoding.
+
+## Tips for Intel Gen9 and Gen11+ when using VAAPI or QSV on Linux
+
+The Intel [Guc/Huc firmware](https://01.org/linuxgraphics/downloads/firmware) must be enabled for optional Low-Power encoding (pre-Gen11 only supports LP H.264).
+
+Instructions for ArchLinux: [Arch Wiki](https://wiki.archlinux.org/title/intel_graphics#Enable_GuC_/_HuC_firmware_loading) and for Debian/Ubuntu: [Brainiarc7's gist](https://gist.github.com/Brainiarc7/aa43570f512906e882ad6cdd835efe57)
+
+For Jasper Lake chips (such as N5095 and N6005), Low-Power encoding must be enabled. But the linux-firmware support is not ready in Ubuntu 20.04.3 LTS. The 21.10 or newer distro is required.
+
 ## Supported Acceleration Methods
 
+Since Jellyfin 10.8 release, the full hardware filtering (scale, de-interlacle, tone-mapping and subtitle burn-in) on Intel, AMD and NVIDIA have been verified with `jellyfin-ffmpeg` 4.4.1.
```

Comment:
I think `since` can confuse people in the future in thinking that lets say, 10.9 has been tested with `jellyfin-ffmpeg` 4.4.1.
```suggestion
In Jellyfin 10.8, full hardware-accelerated filtering (scaling, deinterlacing, tone mapping and subtitle burn-in) on Intel, AMD and NVIDIA hardware have been verified with `jellyfin-ffmpeg` 4.4.1.
```